.github/workflows/publish.yml

```
@@ -32,7 +33,10 @@ jobs:
           if ! git diff --exit-code docs/antora.yml; then
             git config user.name "${GITHUB_ACTOR}"
             git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+            git checkout -b switch-docs-version
             git add docs/antora.yml
             git commit -m "Switch docs version back"
-            git push
+            git push -u origin switch-docs-version
+            gh pr create --title "Switch docs version back" --body "" \
+              --head master -H switch-docs-version
```

`-H` and `--head` are aliases for the same option. I think you meant
```suggestion
              --head switch-docs-version
```